### ISSUE TITLE
LeaderLine on stackblitz

### ISSUE THREAD

**Issue by krishsub2011** [anseki/leader-line#184]
Hello Anseki, 
This is not a bug. I dont know where to put this information, is there a general discussion board. 
I would like to thank you for creating this package and draggable..both excellent stuff.. now to the main issue.

I was trying to work out a simple leader line example on stackblitz environment. 
After some struggle it went through. 
The project is @krishsub2011/leaderline-working for reference. 
Link is https://stackblitz.com/edit/leaderline-working?file=index.js
If you do get the time please check the index.js file, its basically a copy of leader-line-min.js with some additions 
  a) adds the first 13 lines and  line 17-21
  b) defines a function "makeLine" which gets called in the index.html on the button click. 
Can you comment on whether this is the best way to do it ? 
All others failed for me, but I am a newbie to JS. 
regards,
Krishnan

**Comment by anseki** [comment]
Hi @krishsub2011, thank you for the comment.
I can't decide  whether that is the best way or not because I don't know your goal.
If that is a result expected by you, the way is your expected  solution.

**Comment by krishsub2011** [comment]
Hi Anseki,

The final goal was to include your library in an pure angular (ts) based
program.

a) This was after I had learnt how to use leader-line and draggable in a
pure JS environment locally on my machine in an apache-tomcat environment
(this worked well for me).
b) Because I was using stackblitz for angular program development I tried
using leader-line&draggable on an angular project
    on stackblitz. This didnt work.
c) Then I tried to get leader-line working with a pure JS program on
stackblitz which also was not too straightforward and hence
   the solution I posted.

Also I didnt expect you to respond to questions else I would have posted
them earlier -:).

So the question I have for you is,
  a) can leader-line & draggable work on pure angular program ? If so how ?
Examples would be useful.
  b) Also can this solution be made to work on stackblitz because that
seems to be good environment for quick&dirty
      prototyping & testing.

regards,
Krishnan
